src/functional_test_runner/lib/test_loader/create_fake_mocha_ui.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TaskCollector } from './task_collector';
+
+type SuiteFn = () => void;
+type TestFn = () => Promise<void> | void;
+
+export function createFakeMochaUi(taskCollector: TaskCollector) {
+  const describe = Object.assign(
+    (name: string, provider: SuiteFn) => {
+      taskCollector.addSuite(name, provider, 'describe', false, false);
+    },
+    {
+      only: (name: string, provider: SuiteFn) => {
+        taskCollector.addSuite(name, provider, 'describe.only', false, true);
+      },
+      skip: (name: string, provider: SuiteFn) => {
+        taskCollector.addSuite(name, provider, 'describe.skip', true, false);
+      },
+    }
+  );
+
+  const it = Object.assign(
+    (name: string, fn: TestFn) => {
```

Comment:
@spencer will you add the parameter like `tag` to pass smth like `chrome only` that we discussed before

Reply:
That's the plan, I've currently added it in the `it().tag()` format, but I forgot that we planned on adding it as an argument